### PR TITLE
Update parish to county

### PR DESCRIPTION
**Pull request by bseeger**
* changes parish to county

### CODE REVIEW

File: src/main/java/org/mdbenefits/app/data/enums/Counties.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package org.mdbenefits.app.data.enums;
+
+public enum Counties {
+  ALLEGANY("Allegany"),
```
Comment (vrajmohan) [inline src/main/java/org/mdbenefits/app/data/enums/Counties.java:4]: Minor - I would think (this is a question for Programs) that they would want the suffix County for all except Baltimore City:
1. Due to the fact that Baltimore City and Baltimore County are _both_ counties
2. Due to the fact that the counties with a possessive name sound weird without the suffix - e.g., Prince George's

File: src/main/java/org/mdbenefits/app/utils/SubmissionUtilities.java [hunk 2]
```diff
@@ -89,7 +90,7 @@ public static String formatMoney(Double value) {
     }
 
     public static boolean isNolaParish(Submission submission) {
-        return Parish.ORLEANS.name().equals(submission.getInputData().get("parish"));
```
Comment (vrajmohan) [inline src/main/java/org/mdbenefits/app/utils/SubmissionUtilities.java:92]: Can't we get rid of the calls to this method easily?

**Review by vrajmohan — APPROVED** [review]
Minor comments

File: src/main/java/org/mdbenefits/app/data/enums/Counties.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  ANNE_ARUNDEL("Anne Arundel"),
+  BALTIMORE_CITY("Baltimore City"),
+  BALITMORE("Baltimore"),
```
Comment (bseeger) [inline src/main/java/org/mdbenefits/app/data/enums/Counties.java:7]: Oh! Typo! I'll fix this soon.

File: src/main/java/org/mdbenefits/app/data/enums/Counties.java (hunk 1 above)
Comment (bseeger) [reply src/main/java/org/mdbenefits/app/data/enums/Counties.java:4]: Yeah, I thought about this as well. The only annoying part is if we have to remove the word "County" when filling out forms.  Let's see what Programs thinks.

Comment (bseeger) [reply src/main/java/org/mdbenefits/app/data/enums/Counties.java:4]: I can add "County" in and then we can take it out, if need be.

File: src/main/java/org/mdbenefits/app/utils/SubmissionUtilities.java (hunk 2 above)
Comment (bseeger) [reply src/main/java/org/mdbenefits/app/utils/SubmissionUtilities.java:92]: I'll see if it opens up a rabbit hole. If not, I'll take it and calls to it out.

File: src/main/java/org/mdbenefits/app/data/enums/Counties.java (hunk 1 above)
Comment (vrajmohan) [reply src/main/java/org/mdbenefits/app/data/enums/Counties.java:4]: IIRC, this doesn't go on a form.

**Pull request merged**